benchmark-service/benchmark/static/js/autocomplete.js

```diff
@@ -0,0 +1,142 @@
+function auto_complete(place_holder, str_selector) {
```

Comment:
`str_selector` gives me flashbacks to [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation). Let's call it `input_selector` which is less about argument types and more about its purpose. I think in this context its clear it should be a string representation of a [CSS Selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).

Reply:
And, I think this is a great resolution of the duplication, good work!